packages/theme/README.md

```diff
@@ -54,6 +54,23 @@ import '@spectrum-web-components/theme/express/scale-medium.js';
 import '@spectrum-web-components/theme/express/scale-large.js';
 ```
 
+## Example
+
+An `<sp-theme>` element expects a value for each of its `theme`, `color`, and `scale` attributes to either be provided on the element.
```

Comment:
There's only a default _if_ fragments in those spaces have been loaded; default just means first loaded. This means that nested use doesn't need to be so strict, but it doesn't ensure a delivery for you. If you've not opted into one you may be getting functionality that on accident or different than what you expected. The structure here is about preventing the number of KBs that a product needs to ship over the wire.

In that way, you'll actually see the documentation site trigger this warning by default. However, in that case, we load the default styles in CSS so that they are not needed until a user chooses to customize their theme beyond size, prefers color, and Spectrum Classic.